PlayRho/Collision/WorldManifold.hpp

```diff
@@ -118,7 +118,7 @@ class WorldManifold
     ///
     size_type GetPointCount() const noexcept
     {
-        return (IsValid(m_separations[0])? 1: 0) + (IsValid(m_separations[1])? 1: 0);
+        return static_cast<size_type> ((IsValid(m_separations[0])? 1: 0) + (IsValid(m_separations[1])? 1: 0));
```

Comment:
Mea culpa! I mean I don't know why I wasn't using a line more like the following here:
```
return (IsValid(m_separations[0])? 1u: 0u) + (IsValid(m_separations[1])? 1u: 0u);
```

I'd prefer that to casting to the hardcoded `size_type`.

Reply:
g++ wouldn't compile it with my syntax above though. But it did with yours!

Problem I realized was for the conversions warning which was seeing my code promoting the results to a bigger type and then narrowing it implicitly. The cast however avoids that in less syntax than `return (IsValid(m_separations[0])? size_type(1u): size_type(0u)) + (IsValid(m_separations[1])? size_type(1u): size_type(0u));` or using a literal that types it to `size_type` (which is like an 8-bit unsigned integer).